blocks/links-bar-block/features/links-bar/default.jsx

```diff
@@ -33,23 +32,26 @@ const LinksBar = ({ customFields: { navigationConfig = {}, ariaLabel } }) => {
 
 	return (
 		<>
-			<nav
+			<Stack
+				className={BLOCK_CLASS_NAME}
+				justification="center"
+				direction="row"
+				as="nav"
 				key={id}
-				className="links-bar"
 				aria-label={ariaLabel || phrases.t("links-bar-block.element-aria-label")}
 			>
-				{menuItems &&
+				{menuItems?.length > 0 &&
```

Comment:
This should be moved up to encompass the whole return.  There's no need to render the stack or `<hr />` if there are no items, right?

Reply:
looks like this needs to move up @malavikakoppula based on the comment. wanted to make sure this made sense

Reply:
updated empty check to hr tag while no data